app/src/main/java/nl/eduvpn/app/fragment/ConnectionStatusFragment.kt

```diff
@@ -220,11 +233,6 @@ class ConnectionStatusFragment : BaseFragment<FragmentConnectionStatusBinding>()
         viewModel.onResume()
     }
 
-    override fun onPause() {
-        super.onPause()
-        viewModel.onPause()
```

Comment:
This doesn't look right. Maybe empty the onPause function of the viewmodel, but if the function exists, it is expected to be called.

Reply:
An empty viewModel.onPause does nothing, right? So calling it does nothing, so it can be removed. super.onPause will be called even after removing this function so I think this function can be removed.

Reply:
I mean that viewModel.onPause() I assume does something as well, maybe in the super class, if there's an onResume() too. If not, then this is OK, although I don't like the mixing of the new style lifecycle observer callbacks with the regular, onPause / onResume. If we start using the new style, we should convert all other usages as well.

Reply:
I converted the callbacks to LiveData so there is no onStart, onResume and onStop in ConnectionStatusFragment anymore.